libs/features/personalization/personalization.js

```diff
@@ -313,6 +313,7 @@ const clearEntLocalStorage = () => {
 
 export const getEntitlements = (() => {
   let ents;
+  let logoutEventSet;
```

Comment:
Should this be outside `getEntitlements` in case the method is called multiple times? Otherwise `logoutEventSet` is reset every time, right?

Reply:
Since this is a IIFE `ents` and `logoutEventSet` are encapsulated by it.  The return value is the function assigned to `getEntitlements`.

This follows the "Revealing Module Pattern":
* https://gist.github.com/zcaceres/bb0eec99c02dda6aac0e041d0d4d7bf2

Reply:
I missed a few extra parenthesis, my bad 🙈